ci/lint.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.18.3
+
+inputs:
+  - name: dp-frontend-homepage-controller
+    path: dp-frontend-homepage-controller
+
+caches:
+  - path: go/
+
+run:
+  path: dp-frontend-homepage-controller/ci/scripts/lint.sh
```

Comment:
Can we make this file match the [template in dp-cli](https://github.com/ONSdigital/dp-cli/blob/main/project_generation/content/templates/base-app/ci/lint.yml.tmpl) please?

Reply:
The only difference I can see with the rest of the files in the same repo is the inclusion of extra path , which I have fixed in update 1. The other changes are same. as the other yml files in the same repo.